<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rattrapage - Gestion de Clients/Commande</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>791</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabClients">
     <attribute name="title">
      <string>Clients</string>
     </attribute>
     <widget class="QLineEdit" name="lineIdClient">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineNomClient">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="linePrenomClient">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEmailClient">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineTelClient">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineAdresseClient">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="ID">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID Client :</string>
      </property>
     </widget>
     <widget class="QLabel" name="Nom">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
     <widget class="QLabel" name="Prenom">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prénom :</string>
      </property>
     </widget>
     <widget class="QLabel" name="Email">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Email :</string>
      </property>
     </widget>
     <widget class="QLabel" name="Telephone">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Téléphone :</string>
      </property>
     </widget>
     <widget class="QLabel" name="Adresse">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnModifierClient">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>410</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSupprimerClient">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>410</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAjouterClient">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>410</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewClients">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>471</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupRechercheClients">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>219</y>
        <width>251</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Recherche / Tri</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom contient :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Email contient :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Téléphone contient :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineRechercheNomClient">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineRechercheEmailClient">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineRechercheTelClient">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>130</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier par :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboTriClient">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>210</y>
         <width>131</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Nom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Prenom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Email</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ordre :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboOrdreClient">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>250</y>
         <width>131</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Croissant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Décroissant</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="btnRechercheClient">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>170</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Recherche</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnStatsClients">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>460</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Statistiques</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExporterClientsPdf">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>460</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Exporter en PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTotalDepenseClient">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>460</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Total dépensé</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFideliteClient">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>460</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Niveau fidélité</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabCommandes">
     <attribute name="title">
      <string>Commandes</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID Commande :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date commande :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Montant total :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Statut :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mode de paiement :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID Client :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineIdCommande">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineMontantCommande">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineIdClientCommande">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateCommande">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboStatutCommande">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>En cours</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Payée</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Annulée</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboModePaiement">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Espèces</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Carte</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Chèque</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Virement</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>En ligne</string>
       </property>
      </item>
     </widget>
     <widget class="QTableView" name="tableViewCommandes">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>471</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAjouterCommande">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>410</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSupprimerCommande">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>410</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnModifierCommande">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>410</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>271</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Recherche / Tri</string>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date min :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID Client :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Mode de paiement :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date max :</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateMinCommande">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateMaxCommande">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboModePaiementFiltre">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>111</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Tous</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Espèces</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Carte</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chèque</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Virement</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>En ligne</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineIdClientFiltreCommande">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>130</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRechercheCommande">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier par :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ordre :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboTriCommande">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>200</y>
         <width>121</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Date</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Montant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Client</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Paiement</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboOrdreCommande">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>230</y>
         <width>121</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Croissant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Décroissant</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="btnAfficherToutesCommandes">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Afficher Tous</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnExporterCommandesPdf">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>480</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Exporter en PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCommandesPourClient">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>480</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Commandes du client saisi</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnStatsCommandes">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>450</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Statistiques</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnChiffreAffairesTotal">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>450</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Chiffre d'affaires total</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRattrappage">
    <property name="title">
     <string>Rattrappage</string>
    </property>
   </widget>
   <addaction name="menuRattrappage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
